### ISSUE TITLE
For Patrick! Not sure I did this Makefile right

### ISSUE THREAD

**Issue by gbozarth-uchicago** [gbozarth-uchicago/invinst-code-review#1]
Not sure this Makefile, or maybe the source code? Is producing the output correctly... the .csv looks off when I then try to use it for my source code in the cleaning folder. @vm-wylbur

**Comment by vm-wylbur** [comment]
cool! do you mean this one: `invinst-code-review/import/Makefile` If so, it looks pretty good. The `clean` rule might have too many tabs. Does it run?

**Comment by gbozarth-uchicago** [comment]
Yes! I mean this one, and it does run, but when I look at the .cvs file in R it doesn't delimitate properly.

**Comment by vm-wylbur** [comment]
great! the bug you're noting is in the `import/src/import.R` file. You specified the pipe `|` delimiter, which I prefer. 

What is the problem you're having? Is it that you cannot read it? If so, I recommend something like 
```
readr::read_delim(files$input, delim='|')
```

or you can save the file as parquet with `arrow::write_parquet()`. We might schedule a discussion of csv vs binary data formats. There are substantial pros and cons of text vs binary formats, and it's good to know when to use each.
